dev-manuals/test-plans/minigames/finite-quiz/u-finite-quiz-1-playthrough.md

```diff
@@ -0,0 +1,40 @@
+# Finite-Quiz: Playthrough (`u.finite-quiz-1`)
+
+Version: V1.0, 2022-08-26
+Author: Martin Lautenschlager, Levi Otterbach
+
+## Description
+
+This use case verifies that a normal run through the game works correctly.
+
+## Precondition
+
+A configuration has been chosen that both the frontend and the backend know about. The backend has stored test data about this configuration.
```

Comment:
```suggestion
A configuration has been chosen that exists in the backend. At least two questions exist in the given configuration.
```
Why backend and **frontend** know about it? I mean just backend needs to know and frontend receives a response.
Also in this test case, the configuration needs at least 2 questions.

Reply:
I think what they meant was "the frontend knows which configuration to send".